packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -603,16 +603,6 @@ class InspectorController extends DisposableController
     _refreshRateLimiter.scheduleRequest();
   }
 
-  bool identicalDiagnosticsNodes(
```

Comment:
Fyi @incendial, why wasn't this method detected as dead code?

Reply:
Good question, will check.

Reply:
I think I was running the command without the `--monorepo` option. And since `inspector_controller` is in library exports, any unused code is skipped.

With the option it shows a bunch of things:
```
lib/src/screens/inspector/inspector_controller.dart:
    ⚠ unused field highlightNodesShownInBothTrees
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:237:3
    ⚠ unused method getTreeType
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:254:3
    ⚠ unused method highlightShowFromNodeInstanceRef
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:285:3
    ⚠ unused method getSubtreeRootValue
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:493:3
    ⚠ unused method getTreeNode
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:574:3
    ⚠ unused method maybeUpdateValueUI
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:578:3
    ⚠ unused method identicalDiagnosticsNodes
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:602:3
    ⚠ unused method treeTypeDisplayName
      at /Users/dimannich/Desktop/code/devtools/packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart:922:3
```

Reply:
And in other files as well.

Can create a PR if the team is ready for another unused code round 🙂.

Reply:
It would be amazing. Thank you!!!

Reply:
Actually, I was running the command on a stale fork, so some of these reports are already removed 😅